pyproject.toml

```diff
@@ -12,6 +12,7 @@ dependencies = [
     "duckdb>=0.5.0",
     "sqlalchemy>=1.3.22",
     "packaging>=21",
+    "greenlet>=2.0.0",
```

Comment:
I think greenlet is only needed when using async mode of sqlalchemy, so if I as a user only want to use the sync mode, it would be great if I could avoid installing this extra dep. So, I think greenlet should be a dev dependency instead?